configuration/external_parameters.rst

```diff
@@ -80,24 +123,51 @@ You can now reference these parameters wherever you need them.
 
         <doctrine:config>
             <doctrine:dbal
-                driver="pdo_mysql"
-                dbname="symfony_project"
-                user="%database.user%"
-                password="%database.password%"
+                host="%env(DATABASE_HOST)%"
             />
         </doctrine:config>
 
     .. code-block:: php
 
         $container->loadFromExtension('doctrine', array(
             'dbal' => array(
-                'driver'   => 'pdo_mysql',
-                'dbname'   => 'symfony_project',
-                'user'     => '%database.user%',
-                'password' => '%database.password%',
+                'host' => '%env(DATABASE_HOST)%',
```

Comment:
Not sure, at least I don't know how to phrase it :) This is explained indirectly a few lines below this one. If someone has any suggestions, please do, otherwise I might leave it asis.